src/averageAges.js

```diff
@@ -20,6 +20,23 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const mens = people.filter(person => person.sex === 'm');
+  const ages = mens.reduce(
+    (sum, person) => sum + (person.died - person.born),
+    0,
+  ) / mens.length;
```

Comment:
можно еще так
```suggestion
  const ages = mens.reduce((sum, person) => (
    sum + (person.died - person.born),
  ), 0) / mens.length;
```
так даже лучше